tools/stress-cluster/services/otelcollector/otel-collector-config.yml

```diff
@@ -0,0 +1,33 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+      grpc:
+ 
+processors:
+  batch:
+ 
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging, azuremonitor]
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheus, azuremonitor]
+    logs:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [azuremonitor]
+
+exporters:
+  prometheus:
```

Comment:
Remove references to the prometheus exporter.

Reply:
Unless we're also configuring a prometheus service client in our stress cluster, or have configured prometheus for stress metrics.

Reply:
No, we're not (and we could always bring this back later).